recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
     number: 201
-    skip: true                # [win]
+    skip: true                # [win and not py35]
```

Comment:
Does it require C++11? If so, `vc14`/VS 2015 is the only acceptable target. Currently that is a function of support Python versions. So Python 3.5 ensure it only builds on that target.